MaterialLib/Fluid/Density/createFluidDensityModel.cpp

```diff
@@ -0,0 +1,60 @@
+/*!
+   \file  createFluidDensityModel.cpp
+   \brief create an instance of a fluid density class.
+
+   \copyright
+    Copyright (c) 2012-2016, OpenGeoSys Community (http://www.opengeosys.org)
+               Distributed under a Modified BSD License.
+               See accompanying file LICENSE.txt or
+               http://www.opengeosys.org/project/license
+*/
+
+#include "createFluidDensityModel.h"
+
+#include "BaseLib/Error.h"
+
+#include "MaterialLib/Fluid/ConstantFluidProperty.h"
+#include "IdealGasLaw.h"
+#include "LinearTemperatureDependentDensity.h"
+#include "LiquidDensity.h"
+
+namespace MaterialLib
+{
+namespace Fluid
+{
+std::unique_ptr<FluidProperty> createFluidDensityModel(BaseLib::ConfigTree const& config)
+{
+    //! \ogs_file_param{material__fluid__density__type}
+    auto const type = config.getConfigParameter<std::string>("type");
+
+    if (type == "constant")
```

Comment:
Sometimes type is in CamelCase, sometimes all lowercase. Choose one. I prefer CamelCase.